lib/lookup.js

```diff
@@ -95,7 +95,7 @@ function resolve(context, next) {
           next(new Error('no-repository-field in package.json'));
           return;
         }
-        const gitHead = rep.master ? null : meta.gitHead;
+        const gitHead = rep.master || rep.ignoreGitHead ? null : meta.gitHead;
```

Comment:
can you add this new option to the documentation in the readme?

Reply:
Done